### PR TITLE
reference: add some explanation about  _tidb_rowid

### PR DESCRIPTION
**Pull request by AilinKid**
### What is changed, added or deleted? (Required)
add some explanation about  _tidb_rowid

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v4.0 changes for now)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: reference/mysql-compatibility.md [hunk 1]
```diff
@@ -64,6 +64,29 @@ TiDB 实现自增 ID 的原理是每个 tidb-server 实例缓存一段 ID 值用
 
 另外，从 TiDB 2.1.18 和 3.0.4 版本开始，TiDB 将通过系统变量 `@@tidb_allow_remove_auto_inc` 控制是否允许通过 `alter table modify` 或 `alter table change` 来移除列的 `AUTO_INCREMENT` 属性，默认是不允许移除。
 
+注意：TiDB 对于没有指定主键的情况下会使用 _tidb_rowid 来标识行，该数值的分配会和 auto_increment 列（如果存在的话) 共用一个分配器。如果指定了 auto_increment 列为主键，则 tidb 会用 auto_increment 列来标识行。因此会有以下的示例情况：
+
+{{< copyable "sql" >}}
```
Comment (TomShawn) [inline reference/mysql-compatibility.md]: 这段示例是输出结果吗？是否需要用户拷贝？

Comment (AilinKid) [reply reference/mysql-compatibility.md]: 是示例结果，可供用户拷贝测试

Comment (TomShawn) [reply reference/mysql-compatibility.md]: + 用户是否需要拷贝`mysql> `进终端呢？
+ 建议将命令和结果相分离，方便用户复制粘贴。

Comment (AilinKid) [reply reference/mysql-compatibility.md]: 是个一个整体流程，分成多个命令（代码块）感觉隔的比较开，我觉的这样应该挺好

**Comment by Hangzhi** [comment]
@yikeke: I'd like to translate this PR.  I learned from the CONTRIBUTING.md that I can contribute to this community by translating the Chinese documents into English.

**Comment by yikeke** [comment]
> @yikeke: I'd like to translate this PR. I learned from the CONTRIBUTING.md that I can contribute to this community by translating the Chinese documents into English.

Sure, welcome! I've assigned this PR to you and you can start your translation. Look forward to your first contributed pull request~ @Hangzhi

**Pull request merged**